azure-pipelines.yml

```diff
@@ -111,3 +114,13 @@ jobs:
             --prepareMachine
             --ci
           displayName: OSX Build
+
+- ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
+  - template: eng\common\templates\post-build\post-build.yml
+    parameters:
+      # Symbol validation isn't being very reliable lately. This should be enabled back
+      # once this issue is resolved: https://github.com/dotnet/arcade/issues/2871
+      enableSymbolValidation: false
+      # Sourcelink validation isn't passing for Arcade due to some regressions. This should be
+      # enabled back once this issue is resolved: https://github.com/dotnet/arcade/issues/2912
+      enableSourceLinkValidation: false
```

Comment:
I believe SourceLinkValidation is working fine for repos that are not Arcade, so you can remove this section.

@JohnTortugo to confirm.

Reply:
It worked fine for iot. I would suggest first to try running an official build without disabling it, and if it fails then we can disable later. Symbol validation does have an issue that I believe @JohnTortugo is investigating, so that one I believe you should keep.